src/lib/bump-stats/types.ts

```diff
@@ -0,0 +1,29 @@
+export enum STATS_GROUP {
+	STUDIO_APP_LAUNCH = 'studio-app-launch-first',
+	STUDIO_APP_LAUNCH_TOTAL = 'studio-app-launch-total',
+	STUDIO_APP_LAUNCH_UNIQUE = 'local-environment-launch-uniques',
+	STUDIO_IMPORT = 'studio-app-import',
+	STUDIO_EXPORT = 'studio-app-export',
+}
+
+export enum STATS_METRIC {
+	SUCCESS = 'success',
+	FAILURE = 'failure',
+	// Export button types
+	FULL_SITE = 'full-site',
+	DATABASE_ONLY = 'database-only',
+	// Importer types
+	JETPACK_IMPORTER = 'JetpackImporter',
+	LOCAL_IMPORTER = 'LocalImporter',
+	SQL_IMPORTER = 'SQLImporter',
+	PLAYGROUND_IMPORTER = 'PlaygroundImporter',
+	WPRESS_IMPORTER = 'WpressImporter',
+	UNKNOWN = 'unknown',
+}
```

Comment:
```suggestion
	DARWIN = 'darwin',
	LINUX = 'linux',
	WINDOWS = 'win32',
}

export function getPlatform(): StatsMetric {
	switch ( process.platform ) {
		case 'darwin':
			return STATS_METRIC.DARWIN;
		case 'linux':
			return STATS_METRIC.LINUX;
		case 'win32':
			return STATS_METRIC.WINDOWS;
		default:
			throw new Error( `Unsupported platform: ${ process.platform }` );
	}
}
```

Something like this would allow us not to export a separate type for `StatsMetric`. Kind of a nit, but would be nice to address, IMO.